config/FCChh_DelphesCard_WithDipole_v00.tcl

```diff
@@ -37,7 +37,7 @@ TowerMerger
 EFlowMerger
 
 PhotonEfficiency
-PhotonEnergySmearing
+#PhotonEnergySmearing

```

Comment:
Thanks Colin, got it!
Cheers, Filip


On Nov 4, 2015, at 7:56 PM, Colin Bernet <notifications@github.com> wrote:

> In config/FCChh_DelphesCard_WithDipole_v00.tcl:
> 
> > @@ -37,7 +37,7 @@ TowerMerger
> >  EFlowMerger
> >  
> >  PhotonEfficiency
> > -PhotonEnergySmearing
> > +#PhotonEnergySmearing
> Hi Filip, It's probably easier if you do it. In fact, you can just append commits to your branch patch-1 and they will appear here automatically. So that's the same PR, you're just modifying it on the fly. Cheers & thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>